push/Push.md

```diff
@@ -23,3 +23,17 @@ This is essential due to a couple of reasons:
 This is also applicable to product Docker images available at [WSO2 Private Docker Registry](https://docker.wso2.com) mainly due to limitations of storage. For each product version, new images are being frequently added with the latest [WSO2 Updates](https://wso2.com/updates) to products and changes to docker resources used. With limitations of storage, not all images being pushed could be retained in the registry for a long time.
 
 As a result, WSO2 best advise the use of your own private container registry to host all the required images in your deployments, so that you have full control over them. In a rollback event, the recovery of previous version images is effortless with zero risks and zero dependence on outside sources.
```

Comment:
Would it be better to mention that the private registry should be a HA deployment since the application deployment process depends on it?.

Reply:
@ThilinaManamgoda No, let's not mention that here since it is an implementation level detail.